src/section-introduction-to-functions.ptx

```diff
@@ -59,6 +59,13 @@
             </image>
         </figure>
 
+        <definition xml:id="definition-relation">
```

Comment:
This doesn't look right. It looks like now "relation is discussed twice. Once with a definition wrapper and once without.